docs/validationtest/README.md

```diff
@@ -47,10 +81,9 @@ This tool is used to create Kyma subscriptions and their corresponding subscribe
 1. Connect to a Kubernetes cluster.
```

Comment:
In line 79, we could write "Remove Validationtest" to distinguish it from "Stop Validationtest".

however, if "Cleanup" is always the last instruction in your docs, ignore my suggestion :)